pythonforandroid/distribution.py

```diff
@@ -92,15 +95,13 @@ def get_distribution(cls, ctx, name=None, recipes=[],
         # 1) Check if any existing dists meet the requirements
         _possible_dists = []
         for dist in possible_dists:
-            if (
-                ndk_api is not None and dist.ndk_api != ndk_api
-            ) or dist.ndk_api is None:
+            if ((ndk_api is not None and dist.ndk_api != ndk_api)
+                or dist.ndk_api is None or dist.archs is None):
                 continue
-            for recipe in recipes:
-                if recipe not in dist.recipes:
-                    break
-            else:
+            if (set(recipes).issubset(set(dist.recipes))
+                and set(archs).issubset(set(dist.archs))):
```

Comment:
Linter was complaining about these lines:
```
pythonforandroid/distribution.py:99:5: E129 visually indented line with same indent as next logical line
pythonforandroid/distribution.py:102:17: E129 visually indented line with same indent as next logical line
```
Could you fix so Travis finish the build in order to check eventual regressions?